Makefile

```diff
@@ -84,6 +93,25 @@ up: init backbone-up backend-up workers-up generate-certs storage-up ip
 
 stop: backbone-stop backend-stop workers-stop backbone-stop storage-stop
 
+status:
+	@for CONTAINER in $(shell docker ps --format '{{.Names}}' -f NAME=${PROJECT_NAME}); do \
+	  docker cp ./scripts/test_container.sh $$CONTAINER:/; \
+	  docker exec $$CONTAINER chmod +x /test_container.sh; \
+	  docker exec $$CONTAINER env | grep -oP 'AMQP|DATABASE' | uniq | while read -r SERVICE; do \
+	  	echo -n "$$CONTAINER \t to $$SERVICE ... " ; \
+		status="$$(docker exec $$CONTAINER /test_container.sh $${SERVICE} && echo 0 || echo 1)"; \
+		if [ $$status = "0" ]; then \
+			echo "\033[32mPass\e[0m"; \
```

Comment:
is it possible to use colors ?
like: https://github.com/media-cloud-ai/startup/blob/master/Makefile#L34
`$(CYAN_COLOR)`